File: InfoClient.java

public class InfoClient {
    public static void main(String args[]) {
		try {
			Info myInfo = (Info) NamingService.getObjectReference("MyInfoImpl:InfoImpl");
			int retVal = myInfo.get_temp("Timisoara");
			System.out.println("get_temp(\"Timisoara\"): "+retVal);
			String retValS = myInfo.get_road_info(2);
			System.out.println("get_road_info(2): "+retValS);
			NamingService.sendStopMessage("MyInfoImpl:InfoImpl");
		} catch (Exception e){
			e.printStackTrace();
			System.out.println(e.getMessage());
		}
    }

}

File: InfoClient.cs
using System;

namespace CevaTema {
    public static class InfoClient {
	   
        public static void Main()
        {
	        const string name = "MyInfoImpl:InfoImpl";
	        try
	        {
		        var myInfo = (IInfo) NamingService.GetObjectReference(name);
		        var retVal1 = myInfo.get_temp("Timisoara");
		        Console.Write("get_temp(\"Timisoara\"): ");
		        Console.WriteLine(retVal1);
		        var retVal2 = myInfo.get_road_info(1);
		        Console.Write("get_road_info(1): ");
		        Console.WriteLine(retVal2);
		        var retVal3 = myInfo.get_road_info(3);
		        Console.Write("get_road_info(3): ");
		        Console.WriteLine(retVal3);
		        NamingService.SendStopMessage();
	        }
	        catch (Exception e) {
		        Console.WriteLine(e.ToString());
	        }
        }
    }
}